packages/fxa-auth-server/docs/swagger/account-api.ts

```diff
@@ -14,7 +14,7 @@ const ACCOUNT_CREATE_POST = {
   description: '/account/create',
   notes: [
     dedent`
-      Creates a user account. The client provides the email address with which this account will be associated and a stretched password. Stretching is detailed on the [**onepw**](https://github.com/mozilla/fxa-auth-server/wiki/onepw-protocol#creating-the-account) wiki page.
+      Creates a user account. The client provides the email address with which this account will be associated and a stretched password. Stretching is detailed on the [onepw](https://github.com/mozilla/fxa-auth-server/wiki/onepw-protocol#creating-the-account) wiki page.
```

Comment:
I _think_ this might be able to link to the docs site instead of the wiki; https://mozilla.github.io/ecosystem-platform/explanation/onepw-protocol#client-side-key-stretching

Not sure why not all the page headings appear in the sidebar (like the `#creating-the-account` header, for example).